### PR TITLE
Catch exceptions setting up config source, and exit

### PR DESCRIPTION
**Pull request by jonmv**
@baldersheim please review and merge.

### CODE REVIEW

File: configutil/src/apps/modelinspect/main.cpp [hunk 1]
```diff
@@ -76,8 +76,20 @@ Application::getSources(void)
...
 }
 
+config::ConfigUri
+Application::getConfigUri()
+{
+    try {
+        return config::ConfigUri::createFromSpec(_cfgId, config::ServerSpec(_specString));
+    }
+    catch (std::exception &e) {
+        std::cerr << "FATAL ERROR: failed to set up model configuration source.\n";
```
Comment (jonmv) [inline configutil/src/apps/modelinspect/main.cpp]: Argh, forgot `e.what()` here.

Comment (jonmv) [reply configutil/src/apps/modelinspect/main.cpp]: There.

File: configutil/src/apps/modelinspect/main.cpp [hunk 2]
```diff
@@ -76,8 +76,20 @@ Application::getSources(void)
...
 }
 
+config::ConfigUri
+Application::getConfigUri()
+{
+    try {
+        return config::ConfigUri::createFromSpec(_cfgId, config::ServerSpec(_specString));
+    }
+    catch (std::exception &e) {
+        std::cerr << "FATAL ERROR: failed to set up model configuration: " << e.what() << "\n";
```
Comment (baldersheim) [inline configutil/src/apps/modelinspect/main.cpp]: use LOG(FATAL,....

**Review by baldersheim — CHANGES_REQUESTED** [review]
Fix logging so we also get it in the vespalog in startard format if run with redirection

File: configutil/src/apps/modelinspect/main.cpp (hunk 2 above)
Comment (jonmv) [inline configutil/src/apps/modelinspect/main.cpp]:
```suggestion
        LOG(fatal, "Failed to set up model configuration source, will exit: %s", e.what());
```

**Review by baldersheim — APPROVED** [review]

**Pull request merged**